src/NumberInput/NumberInput.tsx

```diff
@@ -220,109 +214,115 @@ export const NumberInput: FC<NumberInputProps> = ({
           </Spinner>
         )}
 
-        {trailingIcon && <Icon render={trailingIcon} color="subtext" />}
-      </Content>
-      {error && <ErrorBox>{errorMsg}</ErrorBox>}
-    </Container>
+        {trailingIcon && <StyledIcon render={trailingIcon} color="subtext" />}
+      </Box>
+    </Field>
   )
-}
-
-interface IContainer {
-  hasLabel: boolean
-  hasError: boolean
-}
-
-const Container = styled(Box)<IContainer>`
-  font-family: 'Gordita', san-serif;
-  display: flex;
-  flex-direction: column;
-  height: auto;
-`
+})
 
 interface IInput {
   error: boolean
   disabled: boolean
   outlined?: boolean
   value?: string
+  prefix?: string
 }
 
-const Content = styled.div<IInput>`
+const Input = styled.input<IInput>`
+  font-family: 'Circular', san-serif;
+  border: none;
   border-bottom: 1px solid;
   border-color: ${({ error }) =>
     theme.colors[`${error ? 'error' : 'outline'}`]};
-  display: flex;
-  align-items: center;
-  height: 32px;
+  color: ${({ error }) => theme.colors[`${error ? 'error' : 'secondary'}`]};
+  font-size: 16px;
+  width: 100%;
+  outline: none;
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
```

Comment:
Not part of your PR but `pointer` is a bit weird for an input? Shouldn't we use `text` instead?